java/squeek/veganoption/integration/waila/ProviderJutePlant.java

```diff
@@ -34,9 +35,9 @@ public ItemStack getWailaStack(IWailaDataAccessor accessor, IWailaConfigHandler
 		{
 			float growthValue = ((BlockJutePlant) accessor.getBlock()).getGrowthPercent(accessor.getWorld(), accessor.getPosition(), accessor.getBlockState()) * 100.0F;
 			if (growthValue < 100)
-				toolTip.add(String.format("%s : %.0f %%", I18n.format("hud.msg.growth"), growthValue));
+				toolTip.add(LangHelper.translate("waila.growth", (int) growthValue));
```

Comment:
Fair enough, it's not terribly important. I was just trying to [maintain consistency with default Waila](https://bitbucket.org/ProfMobius/waila/src/2461c46b2090ddb2f17a5356bd75a0f14f105a10/src/main/java/mcp/mobius/waila/addons/vanillamc/HUDHandlerVanilla.java?at=1.9.4&fileviewer=file-view-default#HUDHandlerVanilla.java-337).